frontend/src/modules/Survey/Components/StepBegin.tsx

```diff
@@ -98,48 +46,39 @@ export const StepBegin = ({
           <StyledWelcomeText>Find study partners!</StyledWelcomeText>
         </StyledTitleWrapper>
         <StyledFields>
-          <StyledTextFieldWrapper
-            style={{
-              marginBottom: calculatePadding(errorEnum.NAME) + 'rem',
-            }}
-          >
-            <NameField
-              containerStyle={textContainerStyle}
-              inputStyle={nameTextInputStyle}
-              value={name}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                setName(e.target.value)
-              }
-              // TODO: add extra thing in conditional so that when they type something that is valid, the error goes away
-              error={
-                error === errorEnum.NAME || error === errorEnum.BOTH
-                  ? error
-                  : ''
-              }
-            />
-          </StyledTextFieldWrapper>
-          <StyledTextFieldWrapper
-            style={{
-              marginBottom: calculatePadding(errorEnum.EMAIL) + 'rem',
-            }}
-          >
-            <EmailField
-              containerStyle={textContainerStyle}
-              inputStyle={emailTextInputStyle}
-              value={email}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                setEmail(e.target.value)
-              }
-              error={
-                error === errorEnum.EMAIL || error === errorEnum.BOTH
-                  ? error
-                  : ''
-              }
-            />
-          </StyledTextFieldWrapper>
+          <TextField
+            placeholder="Name"
+            variant="standard"
+            sx={textInputStyle}
+            value={name}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              setName(e.target.value)
+            }
+          />
+          <TextField
+            value={email}
+            sx={textInputStyle}
+            variant="standard"
+            placeholder="Cornell NetID"
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              setEmail(e.target.value)
+            }
```

Comment:
I think we should keep the placeholder as Cornell Email or something, because internally the rest of the code expects emails (especially relevant to sending emails). Unless we want to change to using NetID throughout, which on the dev side at least involves more work.

Reply:
I could change it so that "@cornell.edu" is appended to the the end of the target value before setEmail is called. Or is it better just to have it be an email field?

Reply:
Idk